airflow/bin/cli.py

```diff
@@ -857,21 +857,31 @@ def webserver(args):
     if ssl_cert and not ssl_key:
         raise AirflowException(
             'An SSL key must also be provided for use with ' + ssl_cert)
+    flask_web_config = conf.getsection('flask_web')
+    if flask_web_config:
+        flask_web_upper_update = {}
+        for k in flask_web_config:
+            K = k.upper()
+            if k != K:
+                flask_web_upper_update[k] = K
+        for k in flask_web_upper_update:
+            flask_web_config[flask_web_upper_update[k]] = flask_web_config[k]
+            del flask_web_config[k]
```

Comment:
This whole block is a little bit hard to follow. What is it doing? What is the purpose.

I suspect that https://github.com/apache/airflow/pull/5379 might make much of this un-needed too.

Reply:
The flask config names are upper-cased (http://flask.pocoo.org/docs/1.0/config), but the airflow ones are lower-cased. When you get variables set by environment (same as many other airflow configs), you want these to operate in the correct way too. Unfortunately the handler for environment in config has already lower-cased them by this point.